src/main/Core/ImageGenerator/Linux/SupportedDesktopEnvironments.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * A list of Linux desktop environments supported by ImageGenerator.
+ */
+export const SupportedLinuxDesktopEnvironment = ["Cinnamon", "GNOME", "KDE", "MATE", "XFCE", "Pantheon"];
```

Comment:
Please use uppercase names only for classes, interfaces or types. And why not using the "LinuxDesktopEnvironment" type here?

```suggestion
export const supportedLinuxDesktopEnvironment: LinuxDesktopEnvironment[] = ["Cinnamon", "GNOME", "KDE", "MATE", "XFCE", "Pantheon"];
```

Reply:
I made the array to keep a list of supported desktops in one place (since we need it in `ImageGeneratorModule.ts` and `LinuxAppIconExtractor.ts`) to ensure better type safety. I updated it in the new commit but using Partial for `iconThemeNameMap` doesn't give as good type safety.

Reply:
Having the array of supported linux desktop environments in one place makes sense here, I didn't see any problem with that.